test/util/date-time.js

```diff
@@ -40,12 +40,16 @@ afterEach(() => {
 // Returns an ISO string for a date in the past that is no earlier than the
 // specified dates.
 export const fakePastDate = (dateStrings) => {
-  const dateTimes = dateStrings
+  const parsed = dateStrings
     .filter(s => s != null)
-    .map(s => DateTime.fromISO(s));
```

Comment:
I like `DateTime`/Luxon in general, but I've removed it from this function. `DateTime` isn't needed: `Date.parse()` is enough here. I sort of think that `DateTime` just added an extra layer rather than simplifying the code.